apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/data_updater/impl.ex

```diff
@@ -909,6 +909,7 @@ defmodule Astarte.DataUpdaterPlant.DataUpdater.Impl do
           {:halt, {:error, reason}}
 
         :error ->
+          Logger.warn("unexpected key #{inspect(key)} in object #{inspect(object)}.")
```

Comment:
I think our habit is using uppercase for the first log line character. Also this Logger.warn has no tag, is it correct?